Doc/library/os.rst

```diff
@@ -1164,7 +1164,7 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    descriptors are :ref:`non-inheritable <fd_inheritance>`. For a (slightly) more
    portable approach, use the :mod:`pty` module.
 
-   .. availability:: some flavors of Unix.
+   .. availability:: Unix.
```

Comment:
This is losing information. I don't know if it's deliberate here.
Would it be possible to use `Unix (some flavors)`?
Or `Unix*`, meaning “see note below” for humans and “don't fail the test” for future tests?

Reply:
It is deliberate. Our documentation already states that “Availability: Unix” means "some Unix but not all Unix-like platforms":

> An “Availability: Unix” note means that this function is commonly found on Unix systems.